src/main/java/com/smartling/marketo/sdk/domain/landingpage/LandingPageTextContentItem.java

```diff
@@ -1,6 +1,9 @@
 package com.smartling.marketo.sdk.domain.landingpage;
 
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
 public class LandingPageTextContentItem extends LandingPageContentItem {
+    @JsonDeserialize(using = ContentDeserializer.class)
     private String content;
```

Comment:
@dkBrazz do we translate anything inside this field?

Reply:
Only is it is a string
If it is an object we skip it

Reply:
Ok